lightning_server.cc

```diff
@@ -1,15 +1,13 @@
 #include "config_parser.h"
+#include "request_handlers.h"
```

Comment:
Can we alphabetize the listing of the included headers? This gives a consistent ordering across our files to see when dependencies are missing, etc.